microdnf.spec

```diff
@@ -19,8 +19,10 @@ BuildRequires:  pkgconfig(smartcols)
 BuildRequires:  help2man
 
 Requires:       libdnf%{?_isa} >= %{libdnf_version}
+%if 0{?rhel} > 8 || 0{%fedora}
```

Comment:
Are you sure about `0{%fedora}`? I suggest that `0{?fedora}` is correct.

Reply:
oops - actually the right syntax is `0%{?fedora}`. Fixed.